pkg/services/ngalert/writer/prom.go

```diff
@@ -71,21 +88,90 @@ func PointsFromFrames(name string, t time.Time, frames data.Frames, extraLabels
 	return points, nil
 }
 
+type httpClientProvider interface {
```

Comment:
Can we export this? PrometheusWriter is exported but depends on a non-exported interface

Reply:
Exported 👍